src/TabMenu/spec.js

```diff
@@ -28,6 +28,30 @@ export default [{
    default: '<code>0px</code>',
    description: 'Set spacing between <code>TabMenuButton</code> components.',
    responsive: false
+}, {
+   name: 'buttonTextColor',
+   type: 'String',
+   default: '<code>#888</code>',
+   description: 'Text <code>color</code> of the <code>TabMenuButton</code> when <code>isActive</code> is set to <code>false</code>',
```

Comment:
What happens when it is active for these new props?

Reply:
The button's text when active is the `highlightColor`. I'm still figuring out these conventions to be honest - still seeing what the customization needs are in real-world usage.

Reply:
It is possible that we will make it like `ListMenu` which also has a `activeTextColor` prop